--- origsrc/gmtk-1.0.5b1/src/Makefile.am	2011-07-19 16:50:28.000000000 -0500
+++ src/gmtk-1.0.5b1/src/Makefile.am	2011-12-27 22:16:59.409981300 -0600
@@ -30,9 +30,10 @@ libgmtk_la_SOURCES = \
 	$(GTK_CFLAGS) \
 	$(ALSA_CFLAGS)
 
-libgmtk_la_LDFLAGS = 
+libgmtk_la_LDFLAGS = -no-undefined
 
 libgmtk_la_LIBADD = \
+    libgmlib.la \
     $(GTK_LIBS) \
     $(ALSA_LIBS)
     
@@ -60,7 +61,7 @@ libgmlib_la_CFLAGS = -fPIC \
 	$(ALSA_CFLAGS) \
 	$(PULSE_CFLAGS)	
 
-libgmlib_la_LDFLAGS = 
+libgmlib_la_LDFLAGS = -no-undefined
 
 libgmlib_la_LIBADD = \
 	$(GLIB_LIBS) \
