--- origsrc/gmtk-1.0.5b1/src/Makefile.am	2011-07-19 16:50:28.000000000 -0500
+++ src/gmtk-1.0.5b1/src/Makefile.am	2011-12-27 22:16:59.409981300 -0600
@@ -26,7 +26,7 @@ libgmtk_la_SOURCES = \
 	gmtk_output_combo_box.c \
 	gmtk_output_combo_box.h
 
-libgmtk_la_CFLAGS = -fPIC \
+libgmtk_la_CFLAGS = \
 	$(GTK_CFLAGS) \
 	$(ALSA_CFLAGS)
 
@@ -50,7 +51,7 @@ libgmlib_la_SOURCES = \
 	gm_strfuncs.c \
 	gm_strfuncs.h
 	
-libgmlib_la_CFLAGS = -fPIC \
+libgmlib_la_CFLAGS = \
 	$(GLIB_CFLAGS) \
 	$(GCONF_CFLAGS) \
 	$(GIO_CFLAGS) \
